mycore-iiif/src/main/java/org/mycore/iiif/image/resources/MCRIIIFImageResource.java

```diff
@@ -114,6 +120,8 @@ public Response getInfoRedirect(@PathParam(IMPL_PARAM) String impl,
 
     @GET
     @Path("{" + IDENTIFIER_PARAM + "}/{region}/{size}/{rotation}/{quality}.{format}")
+    @MCRCacheControl(maxAge = @MCRCacheControl.Age(time = 1, unit = TimeUnit.HOURS),
+        sMaxAge = @MCRCacheControl.Age(time = 1, unit = TimeUnit.HOURS))
```

Comment:
How often do images change these days? I would prefer a cache time from at least a week.

Reply:
A file in a derivate in theory can change all the time, so it basically should never be cached with a age. 
If we set the time to a week or longer, then we limit the use cases for the Interface for a small amount of saved bandwidth.